microsoft-edge/extensions-chromium/developer-guide/declare-permissions.md

```diff
@@ -24,27 +24,91 @@ ms.date: 03/17/2021
 
 To use most of the `chrome.*` APIs, your extension must declare its intent in the `permissions` fields of the manifest.  An extension can declare permissions using a permission string from the table that follows, or use a pattern to match similar strings.  Permissions help to constrain your extension if it gets compromised by malware.  Some permissions are displayed to users for their consent before installation of the extension, or at runtime, as needed; these are Permission Warnings.
 
-If an API requires you to declare permissions in the manifest, see the documentation for that API to understand the needed permissions.  For example, the Storage API page describes how to declare the `storage` permission.
+If an API requires you to declare permissions in the manifest, see the documentation for that API to understand the needed permissions.  For example, the [Storage API](https://developer.chrome.com/docs/extensions/reference/storage/) page describes how to declare the `storage` permission.
 
 The following code outlines how to declare permissions in the manifest file:
 
+#### [Manifest V3](#tab/v3)
```

Comment:
Engineering to review code

Reply:
We don't need this because immediately after this, I see "Here's an example of the permissions part of a manifest file" and that covers the same thing again.

Reply:
Confirmed exact dup.  Removed.